src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -126,6 +126,54 @@ public String vin() throws SmartcarException {
     return this.call("permissions", "GET", null, ApplicationPermissions.class).getData().getPermissions();
   }
 
+  /**
+   * Send request to the /permissions endpoint
+   *
+   * @param permission the permission to check on the vehicle
+   *
+   * @return if vehicle has specified permission
+   * @throws SmartcarException if the request is unsuccessful
+   */
+  public boolean hasPermissions(String permission) throws SmartcarException {
+    try {
+      for(String vehiclePermission: this.permissions()){
+        if(vehiclePermission.equals(permission))
+          return true;
+      }
+
+      return false;
+    } catch (SmartcarException exception) {
```

Comment:
We don't need the try/catch block right? the exception is going to be thrown if `this.permissions()` throws

Reply:
We do since we need to catch the exception this.permissions() throws and do something with it.